src/main/resources/jnt_listSites/html/sitesTableRow.jspf

```diff
@@ -130,32 +130,12 @@
                     </td>
                 </c:if>
                 <td>
-                    <c:choose>
-                        <c:when test="${jcr:hasPermission(node, 'fileManager')}">
-                            <a href="<c:url value='/engines/manager.jsp?conf=filemanager&site=${node.identifier}'/>" target="_blank">
-                                <img src="<c:url value='/icons/search.png'/>" width="16" height="16" alt=" "
-                                     role="presentation" style="position:relative;"/>
-                            </a>
-                        </c:when>
-                        <c:otherwise>
-                            <img src="<c:url value='/icons/cancel.png'/>" width="16" height="16" alt=" " role="presentation"
-                                 style="position:relative;"/>
-                        </c:otherwise>
-                    </c:choose>
+                    <img src="<c:url value='/icons/cancel.png'/>" width="16" height="16" alt=" " role="presentation"
+                         style="position:relative;"/>
```

Comment:
What's the point of displaying a column with just a cancel icon ?

Reply:
<img width="1112" alt="Capture d’écran 2020-04-24 à 14 16 33" src="https://user-images.githubusercontent.com/12768472/80211593-490a7700-8636-11ea-939f-610cff80d89c.png">
because if I remove them it will look like this

Reply:
Can't you also remove the headers ?

Reply:
I didn't want to change the UI but ok I'll try to remove the headers